scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -1070,15 +1099,11 @@ BEGIN
       )
       SELECT ', ST_Transform(' 
             || t.missing_srid_start || t.attname || t.missing_srid_end
-            || ',4326)::Geometry(' 
-            || t.geomtype 
-            || ',4326) AS '
+            || ',4326)::Geometry(GEOMETRY,4326) AS '
             || const.geomcol
```

Comment:
Actually I kind of dropped that requirement in this changeset: 010dd13e

But turns out it never fully worked and also there's a problem with the check function calculating the expectations when there's a the_geom but not a the_geom_webmercator.